### ISSUE TITLE
How to get rid of this transparent frame?

### ISSUE THREAD

**Issue by rykugur** [KiKaraage/ArcWTF#53]
This only started recently, after a system update, so unsure if it's something to do with firefox, or perhaps wayland. It happens in both gnome and hyprland.

![image](https://github.com/KiKaraage/ArcWTF/assets/2609898/ab70c3e4-6837-4485-b7a8-ddd8e196570a)

You can see steam through the firefox window.

**Comment by ArchitektApx** [comment]
I can confirm this happens (and causes visual glitches) after updating to Firefox 126 (in gnome/wayland). 
Downgrading to FF 124/125 makes it work again. 

What's strange is that I get heavy glitching (repeating cursor etc. in that area but it won't fully show on screenshots/videos - at least in wayland)

![Screenshot from 2024-05-18 17-40-45](https://github.com/KiKaraage/ArcWTF/assets/21159658/e14e43ef-0c5b-4521-9c6f-05a88cf4ffcc)

**Comment by AXVIII3** [comment]
Same issue here with KDE Wayland on Kinoite
![image](https://github.com/KiKaraage/ArcWTF/assets/76608488/c46f00ee-4238-4a99-a39c-d28fa4095bdc)

**Comment by axviii3** [comment]
> Same issue here with KDE Wayland on Kinoite ![image](https://private-user-images.githubusercontent.com/76608488/332755636-c46f00ee-4238-4a99-a39c-d28fa4095bdc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MjYzMjEsIm5iZiI6MTcxNjgyNjAyMSwicGF0aCI6Ii83NjYwODQ4OC8zMzI3NTU2MzYtYzQ2ZjAwZWUtNDIzOC00YTk5LWEzOWMtZDI4ZmE0MDk1YmRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDE2MDcwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkMjE4MTUxMjlkMDA3M2UxMzUwMzU3NTI4NDY2ZTVkMmRkNTIwNmMzN2I0NDA3YTMyM2UyODI2Y2E4YWVhYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._OsHkIEjNX190IkMTB1PLfDprZAwxA9mABk7hPcKSJA)

I tried setting this up again today and this issue was magically gone for me? I don't know what change I could have made fixed this but here are my thoughts from most probable to least

* Installing a KWin script called Karousel for better tiling with KDE Wayland
* Turning on desktop effects (the ones I turned on were wobble, and all animations)
* Already having firefox color installed with catppuccin macchiato theme

I am really out of ideas why this happened but glad it did

**Comment by rykugur** [comment]
> > Same issue here with KDE Wayland on Kinoite
> 
> I tried setting this up again today and this issue was magically gone for me? I don't know what change I could have made fixed this but here are my thoughts from most probable to least
> 
>     * Changing global theme (and application window style) to Scratchy
> 
>     * Installing a KWin script called Karousel for better tiling with KDE Wayland
> 
>     * Turning on desktop effects (the ones I turned on were wobble, and all animations)
> 
>     * Already having firefox color installed with catppuccin macchiato theme
> 
> 
> I am really out of ideas why this happened but glad it did
> 
> ![image](https://private-user-images.githubusercontent.com/76608488/334162341-d38695ff-97b1-4a53-b2d7-b8549b8c1f5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NjIyMjksIm5iZiI6MTcxNjg2MTkyOSwicGF0aCI6Ii83NjYwODQ4OC8zMzQxNjIzNDEtZDM4Njk1ZmYtOTdiMS00YTUzLWIyZDctYjg1NDliOGMxZjVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDAyMDUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4OTBiYzgzYzhiOTUwYWU1NDcyYTcxZjg1NDU4NmY4OTNhMmYxMThlOWYyNjBlYWI4ZTNhZWJmY2ZlY2MxNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eX_DGqI5Jnc7vgcpuzfGhCw5C7kNHCR8ff9--pGybTk)

Interesting - I switched to Catppuccin on the Firefox theme store and that fixed it. I tested switching back to to one of the built-in themes and was able to reproduce the issue.

**Comment by axviii3** [comment]
> > > Same issue here with KDE Wayland on Kinoite
> > 
> > 
> > I tried setting this up again today and this issue was magically gone for me? I don't know what change I could have made fixed this but here are my thoughts from most probable to least
> > ```
> > * Changing global theme (and application window style) to Scratchy
> > 
> > * Installing a KWin script called Karousel for better tiling with KDE Wayland
> > 
> > * Turning on desktop effects (the ones I turned on were wobble, and all animations)
> > 
> > * Already having firefox color installed with catppuccin macchiato theme
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I am really out of ideas why this happened but glad it did
> > ![image](https://private-user-images.githubusercontent.com/76608488/334162341-d38695ff-97b1-4a53-b2d7-b8549b8c1f5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NjIyMjksIm5iZiI6MTcxNjg2MTkyOSwicGF0aCI6Ii83NjYwODQ4OC8zMzQxNjIzNDEtZDM4Njk1ZmYtOTdiMS00YTUzLWIyZDctYjg1NDliOGMxZjVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDAyMDUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4OTBiYzgzYzhiOTUwYWU1NDcyYTcxZjg1NDU4NmY4OTNhMmYxMThlOWYyNjBlYWI4ZTNhZWJmY2ZlY2MxNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eX_DGqI5Jnc7vgcpuzfGhCw5C7kNHCR8ff9--pGybTk)
> 
> Interesting - I switched to Catppuccin on the Firefox theme store and that fixed it. I tested switching back to to one of the built-in themes and was able to reproduce the issue.

Yeah.. I came back to report this exact observation... I also noted the broken window borders on startup until the theme kicks in... It seems like the default themes break something?

**Comment by GwadaPuk** [comment]
Same for me and setting another theme also solved the problem thanks :)
